docs/topics/auth/passwords.txt

```diff
@@ -600,6 +600,13 @@ Django includes four validators:
     Validates that the password is of a minimum length.
     The minimum length can be customized with the ``min_length`` parameter.
```

Comment:
I think in a separate commit, let's document `get_help_text` for consistency

Reply:
Either, we can have this in a separate PR that gets merged before this one (with Refs #35782)

Reply:
https://github.com/django/django/pull/18673